### PR TITLE
Bluesocket support

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Adtran Bluesocket support

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## New Features
* Bluesocket Adtran controller support.

### CODE REVIEW

**Review by JeGoi — APPROVED** [review]
Looks good for me.
Was not able to find version in MIB, too.

**Comment by lzammit** [comment]
Looks good to me. Missing Documentation.

**Comment by extrafu** [comment]
@fdurand Update the doc with the screenshots.

**Comment by fdurand** [comment]
BUMP

File: docs/network/networkdevice/bluesocket.asciidoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+==== MAC Authentication
+
+===== Bluesocket side
+
+In order to configure mac authentication on the Bluesocket, you must have access to the controller.
+
+First, you must configure a RadiusWebAuthServer in External Authentication and enable "Enable Radius MAC Authentication" and add Authentications rules.
+
+This Authentication Rules needs to match the PacketFence role you define to the Bluesocket role you defined.
```
Comment (JeGoi) [inline docs/network/networkdevice/bluesocket.asciidoc]: This Authentication Rules needs to match with the PacketFence `role` you have defined for the Bluesocket.

File: docs/network/networkdevice/bluesocket.asciidoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+image::bluesocket-radiuswebserver.png[scaledwidth="100%",alt="Bluesocket radius web server"]
+
+Next you have to create an SSID in Wireless -> SSIDs and important, check for "Enable Captive Portal Authentication".
```
Comment (JeGoi) [inline docs/network/networkdevice/bluesocket.asciidoc]: Next, you need to create an SSID in `Wireless -> SSIDs` and important, check for "Enable Captive Portal Authentication".

**Review by JeGoi — COMMENTED** [review]
Minor comments
images look fine

**Comment by extrafu** [comment]
@JeGoi If all is fine, merge.

**Pull request merged**